packages/react-core/src/demos/examples/Tabs/GrayTabs.tsx

```diff
@@ -0,0 +1,158 @@
+import React from 'react';
+import {
+  PageSection,
+  PageSectionVariants,
+  Tabs,
+  Tab,
+  TabContent,
+  TabContentBody,
+  TabTitleText,
+  Title,
+  Grid,
+  GridItem,
+  Flex,
+  FlexItem,
+  Card,
+  CardBody,
+  CardHeader
+} from '@patternfly/react-core';
+import DashboardWrapper from '../../examples/DashboardWrapper';
+
+export const GrayTabsDemo = () => {
```

Comment:
Could update this to be explicit about the return type:
`GrayTabsDemo: React.FunctionComponent = () => {`